src/Symfony/Component/Serializer/Mapping/AttributeMetadataInterface.php

```diff
@@ -31,6 +31,8 @@ public function getName(): string;
 
     /**
      * Adds this attribute to the given group.
+     *
+     * @return void
      */
     public function addGroup(string $group);
```

Comment:
This interface is internal. @nicolas-grekas do you also not want to use the real void type for safe methods?

Reply:
if it's internal, we definitely can add the real return type now

Reply:
I updated the code to use native void return types on internal interfaces